independent-publisher-connectors/Airtable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,490 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Airtable",
+    "description": "Airtable custom connector to read and write data",
+    "version": "1.0",
+    "contact": {
+      "name": "Woong Choi",
+      "email": "Woong.Choi@sevensigma.com.au"
+    }
+  },
+  "host": "api.airtable.com",
+  "basePath": "/v0",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{Base ID}/{Table}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "records": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Record ID"
+                      },
+                      "createdTime": {
+                        "type": "string",
+                        "description": "Record Created Time"
+                      }
+                    }
+                  },
+                  "description": "Records"
+                }
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "Base ID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Base ID",
+            "x-ms-summary": "Base ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Table",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Table",
+            "x-ms-summary": "Table",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "filterByFormula",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "description": "filterByFormula",
```

Comment:
@last72 Can you please provide valid description value for the property as it seems like an operation id.

Documentation - [Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar description values in the swagger.